test/unit/services/content_item_publisher_test.rb

```diff
@@ -1,25 +1,51 @@
 require "test_helper"
+require "gds_api/test_helpers/publishing_api"
 
 class ContentItemPublisherTest < ActiveSupport::TestCase
+  include GdsApi::TestHelpers::PublishingApi
+
   setup do
     load_path = fixture_file("smart_answer_flows")
     SmartAnswer::FlowRegistry.stubs(:instance).returns(stub("Flow registry", find: @flow, load_path: load_path))
   end
 
-  test "sending item to content store" do
-    start_page_draft_request = stub_request(:put, "https://publishing-api.test.gov.uk/v2/content/3e6f33b8-0723-4dd5-94a2-cab06f23a685")
-    start_page_publishing_request = stub_request(:post, "https://publishing-api.test.gov.uk/v2/content/3e6f33b8-0723-4dd5-94a2-cab06f23a685/publish")
-    flow_draft_request = stub_request(:put, "https://publishing-api.test.gov.uk/v2/content/154829ba-ad5d-4dad-b11b-2908b7bec399")
-    flow_publishing_request = stub_request(:post, "https://publishing-api.test.gov.uk/v2/content/154829ba-ad5d-4dad-b11b-2908b7bec399/publish")
+  context "#sync" do
+    setup do
+      start_page_content_id = SecureRandom.uuid
+      flow_content_id = SecureRandom.uuid
+      @start_page_draft_request = stub_publishing_api_put_content(start_page_content_id, {})
+      @start_page_publishing_request = stub_publishing_api_publish(start_page_content_id, {})
+      @flow_draft_request = stub_publishing_api_put_content(flow_content_id, {})
+      @flow_publishing_request = stub_publishing_api_publish(flow_content_id, {})
+      @flow = stub("flow",
+                   name: "bridge-of-death",
```

Comment:
What inspired this?

Reply:
Haha it was the existing the test data unfortunately not my imagination.